stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -574,7 +574,7 @@ public void onClose(Status status, Metadata trailers) {
     }
   }
 
-  private static final class ThreadlessExecutor implements Executor {
+  private static final class ThreadlessExecutor implements SerializingExecutor {
```

Comment:
I'm a bit uncomfortable implementing something from internal and then expecting a different module to do instanceof. I guess it doesn't hurt anything if the instanceof fails, but currently we only use internal classes within a module; we don't pass `.internal` things between packages.